### DISCUSSION TITLE
DefaultHeartBeatSyncerTopic need be created manually?

### DISCUSSION THREAD

**Discussion by zuoHx580** [apache/rocketmq#6834] [Q&A]
version：5.1.0
先启动namesrv集群，后启动broker，最后启动proxy，提示：create system broadcast topic DefaultHeartBeatSyncerTopic failed on cluster DefaultCluster

proxy.log如下
![image](https://github.com/apache/rocketmq/assets/45234194/8a0a8025-690e-43eb-8108-b4348fe241bf)

broker.log
![image](https://github.com/apache/rocketmq/assets/45234194/6664440a-0fe7-4d46-a949-fa690e93647b)

broker_default.log
![image](https://github.com/apache/rocketmq/assets/45234194/2e897cfa-c781-42a7-8585-213cacc57024)

**Comment by ShadowySpirits** [comment]
需要配置 rocketMQClusterName，proxy 启动时会向指定的集群创建用于心跳同步的 topic

**Reply by zuoHx580** [reply]
指定的配置里有配置rocketMQClusterName。
![image](https://github.com/apache/rocketmq/assets/45234194/8abdda86-fa5e-4e87-b4b6-74442bb2a66b)

**Comment by zuoHx580** [comment]
确实也没这个topic

![image](https://github.com/apache/rocketmq/assets/45234194/f9e593c7-1464-48be-a7d2-a3d59c78b0a7)
![image](https://github.com/apache/rocketmq/assets/45234194/7e71756d-66d0-4aa0-aa36-56cffd7f7492)

**Comment by ipcpu** [comment]
+1
